charts/dify/templates/deployment.yaml

```diff
@@ -250,7 +253,10 @@ spec:
             httpGet:
               path: /apps
               port: http
-            initialDelaySeconds: 1
+              httpHeaders:
+              - name: accept-language
+                value: en
+            initialDelaySeconds: 30
```

Comment:
The initialDelaySeconds of 30 seconds is too long for the frontend's readinessProbe. Next.js starts up very quickly, usually completing within a few seconds. With initialDelaySeconds set to 30, it takes at least 30 seconds for the frontend to become available.  reduce the initialDelaySeconds to 3 seconds?

Reply:
How about make this configurable? I experienced unavailability for at least 20 seconds

Reply:
In my case, the langgenius/dify-web container took less than one second to reach readiness after starting.

```console
2024-04-24T06:17:49: PM2 log: Launching in no daemon mode
......
06:17:50 0|dify-web  |  ✓ Ready in 210ms
```

It's important to note that in Kubernetes, the time taken for container scheduling and image pulling is not included in the health check calculations.